04_environment/index.tex

```diff
@@ -1 +1,400 @@
-\chapter{Game Environment}
\ No newline at end of file
+\chapter{Environment}
+\section{Foraging}
+\subsection{Foraging Background}
+
+Foraging is a base requirement that is needed as it is a method of obtaining more resources from the starting amount.Foraging is based on the concept of inputting resources to get a return.\\
+
+\subsubsection{Underlying concept}
+The goal of foraging in this “game” is to introduce a certain dilemma to the agents to further understand the behaviours of multiple (6) intelligent agents working together to reach a goal with different personal goals. By having 2 methods of foraging it introduces the concept of decision making between the islands allowing for strategic interactions to maximise personal benefits whilst still reaching the overarching goal of survival.\\  
+
+The inspiration for our concept of foraging comes from the stag hunt game furthermore it introduces elements from it. Specifically it brings in the concept of \textbf{no dominant strategy} and how \textbf{ one agents choice can impact other agents.}\\
```

Comment:
bad sentence -- revise

Reply:
comma after "Specifically"...

suggested re-writing:
**Specifically,** ...

Reply:
reference to or use of personal pronouns or language such as "our" is generally frowned upon in academic writing in the UK

Reply:
the point is in bold is very strong though. Suggested rewrite: 

Specifically, this approach results in a game with \textbf{no dominant strategy} and in which \textbf{ one agents choice can impact another agent's.}\\

Reply:
full stop should be out of the bold (textbf) brackets